main/filtering.js

```diff
@@ -76,11 +76,71 @@ function requestDomainIsException (domain) {
   return enabledFilteringOptions.exceptionDomains.includes(domain)
 }
 
+function removeTrackingParams (url) {
+  const globalParamsToRemove = [
+    // microsoft
+    'msclkid',
+    // google
+    'gclid',
+    'dclid',
+    // facebook
+    'fbclid',
+    // yandex
+    'yclid'
+  ]
+  const siteParamsToRemove = {
+    'www.amazon.com': [
+      '_ref',
+      'ref_',
+      'pd_rd_r',
+      'pd_rd_w',
+      'pf_rd_i',
+      'pf_rd_m',
+      'pf_rd_p',
+      'pf_rd_r',
+      'pf_rd_s',
+      'pf_rd_t',
+      'pd_rd_wg'
+    ],
+    'www.ebay.com': [
+      '_trkparms'
+    ]
+  }
```

Comment:
We can have these at the beginning of the file as it will be easier to find, what do you think? Also, [I wrote a userscript](https://github.com/lubiedo/min-userscripts/blob/main/utmcleaner.js) for these that had a similar purpose, you might find some other parameter that you could use.